components/tooltip/tooltip.js

```diff
@@ -14,12 +15,19 @@ const pointerOverhang = 7; /* how far the pointer extends outside the content */
 const pointerRotatedLength = Math.SQRT2 * pointerLength;
 const pointerRotatedOverhang = ((pointerRotatedLength - pointerLength) / 2) + pointerOverhang;
 
-const pointerGap = 6; /* spacing between pointer and target */
+const pointerGap = 0; /* spacing between pointer and target */
 const defaultViewportMargin = 18;
 const contentBorderRadius = 6;
 const contentBorderSize = 1;
 const contentHorizontalPadding = 15;
 
+const interactiveElements = {
```

Comment:
`a` wasn't included in this list because there is inconsistent behavior across screenreaders if no `href` is set.

Reply:
Would it be worth detecting that as a special case inside `_isInteractive`? i.e. the tag name is `a` AND it has an `href` set?

Reply:
I considered that, the reason I didn't is because the `announce` works almost as good and I thought there might be some cases where the `href` attribute is dynamically added or removed which wouldn't be accounted for. I'm on the fence so I'm certainly open to adding that as a special case.

Reply:
Cool, don't feel strongly. Could mitigate the dynamic modification by doing the `_isInteractive` call at runtime (which you're already doing I think).

Reply:
The one issue there is that I'd need to also dynamically remove `aria-describedby` since JAWS will announce anchor tags with no `href` but NVDA won't. For now I'm going to stick with `announce` for `a` tags which I think should work pretty good but if issues do arise then I'll explore adding some special conditions for `a` with and without `href`.